app/models/library_hours.rb

```diff
@@ -19,6 +18,10 @@ def business_days
     end.compact
   end
 
+  def library
+    config.scanning_library_proxy[@library] || @library
```

Comment:
I'm not thrilled by this. I wonder if we could teach the `config.hours_api_location_map` hash how to alias the hours for us..